### ISSUE TITLE
[469b] replace invalid player name characters

### ISSUE THREAD

**Issue by SeriousBarbie** [OldUnreal/UnrealTournamentPatches#315]
Looks like if a joining player has whatever invalid characters in his name, it is not displayed (or as blanks). 469b server's log:

> Level server received: LOGIN RESPONSE=776337779 URL=MH-Andromeda-Ancient-Test1V?Name=D?los?Class=BotPack.TMale2?Team=0?skin=SoldierSkins.hkil?Face=SoldierSkins.Matrix?Voice=BotPack.VoiceMaleTwo?OverrideClass=?Checksum=NoChecksum
> Client passed challenge
> Login request: MH-Andromeda-Ancient-Test1V?Name=D?los?Class=BotPack.TMale2?Team=0?skin=SoldierSkins.hkil?Face=SoldierSkins.Matrix?Voice=BotPack.VoiceMaleTwo?OverrideClass=?Checksum=NoChecksum
> Level server received: NETSPEED 10000

The question mark in "Name=D?los" is in fact a "?" (ASCII 3Fh) in server's log which is not in UTF8 but plain ASCII format as I can see.

(I would like to add a client's screenshot but the browser shows the message _Something went really wrong, and we can't process that file_) -.-
In clients that name is displayed as a number of (5?) blanks.

**Comment by SeriousBuggie** [comment]
![img](https://user-images.githubusercontent.com/70026933/114438378-58305f00-9bd0-11eb-8284-b5e82dc9eb2c.png)
![img](https://user-images.githubusercontent.com/70026933/114438481-7ac27800-9bd0-11eb-869c-6a82c071a154.png)

**Comment by SeriousBuggie** [comment]
Also this break server info:
![img](https://user-images.githubusercontent.com/70026933/114438749-d4c33d80-9bd0-11eb-98f1-11babb1cc3fd.png)
![img](https://user-images.githubusercontent.com/70026933/114438791-e1e02c80-9bd0-11eb-8160-495324226f8f.png)
https://www.gametracker.com/server_info/81.169.240.101:7777/

**Comment by an-eternity** [comment]
Never faced this before... Need to add new rules in "Login()" or "PreLogin()" of 'GameInfo' to handle this case...
Wonder, if same problem would happen with 3Dh, and probably with 2Fh and 5Ch (and maybe with some else symbols)...

**Comment by stijn-volckaert** [comment]
Technically, the game should be able to handle any unicode character, but the font textures currently only contain ISO-8859-1 glyphs. What this means is that any character that is not on the ISO-8859-1 page will show up as a blank space on the scoreboard, but internally the correct name should be stored. The UTStats log should also contain the correct name.

Something clearly did go wrong here though, so we'll have to do some debugging. Could you upload your server log (and utstats log if you have it)?

**Comment by SeriousBarbie** [comment]
Looks like it was that name:
![Unreal ngLog 2021 04 12 18 23 36 log](https://user-images.githubusercontent.com/81397053/114755482-f7cd2900-9d59-11eb-9122-111bb312a58c.jpg)

> Could you upload your server log

Is there a possibility to keep that secret? Because here in DE already an IP address is below the GDPR.

**Comment by Snuggx** [comment]
I think I can clear this up.
I'm sure it was me.
I have only been active again since 2020 and use a 15 year old User.ini that is still configured as admin for our Assault servers.
I used this "feature" when I went to the server as a spectator.
Since I have almost only played on our Assault Server since 2020, and an Auth system is installed on these servers, which automatically sets the name registered in the forum, I didn't think about it when I started playing on MH servers. 

At some point I noticed this, and I've been using Che since then :) 

From my INI:  3=name Δῆλος

Copied from there:

https://de.wikipedia.org/wiki/Delos
